[Improve] Enhance complement function transformation (#10376)
apache/dolphinscheduler: docs/docs/en/about/glossary.md

failure. **End** means that once a failed task is found, Kill will also run the parallel task at the same time, and the
process fails and ends

> **Complement**: Supplement historical data，Supports **interval parallel and serial** two complement methods, and supports **select date range and manually enter date** two date methods

Reviewer: ```suggestion
**Complement**: Supplement historical data，supports **interval parallel and serial** two complement methods, and two types of date selection which include **date range** and **date enumeration **.
```

> **Complement**: Supplement historical data，supports **interval parallel and serial** two complement methods, and two types of date selection which include **date range** and **date enumeration **.

Reviewer: ```suggestion
**Complement**: Supplement historical data，supports **interval parallel and serial** two complement methods, and two types of date selection which include **date range** and **date enumeration**.
```
